### ISSUE TITLE
`rotate`s do not appear to work.

### ISSUE THREAD

**Issue by kyleshevlin** [LegendApp/legend-motion#5]
I've tried making the simplest example with `rotate` and can't get to the bottom of why it's not working.

`rotate`* types expect a number, though I'm not sure what that number maps to. Degrees? Radians?

```javascript
<Motion.View
  animate={{ rotate: 90 }}
  initial={{ rotate: 0 }}
  transition={{
    type: "timing",
    duration: 2000
  }}
  style={{ borderWidth: 1, height: 50, width: 50 }}
/>
```

Results in a box that just sits there, statically. I can verify that other transforms, like `x` and `y` work, but not any of the rotations.

**Comment by kyleshevlin** [comment]
Things that may matter:

- using `react-native-web`
- using this for components that run on native _and_ the browser

**Comment by jmeistrich** [comment]
Thanks for catching that! Fixed in 2.1.8.

You're right, it should not be a number. It should match the Animated values, so `90deg` or `1.57rad`. It was previously mis-typed as number and not passing interpolated values into transforms. So now it should interpolate the `deg` and `rad` strings correctly.

**Comment by kyleshevlin** [comment]
That's great. I didn't get a chance to add an issue, but my colleague and I noticed that `skew`s also did not appear to do anything. I'll give 2.1.8 a try today and see if that has resolved as well.

**Comment by jmeistrich** [comment]
Yes, `skew`s should also be fixed by that change too. I just released a 2.1.9 that fixes the types for `skew`s.

**Comment by kyleshevlin** [comment]
Thank you for the quick responses. It is appreciated!
